### ISSUE TITLE
List out features added to the original fork/project

### ISSUE THREAD

**Issue by tyliec** [CodeWithAloha/uipa#52]

**Comment by tyliec** [comment]
One feature that was added on top of the original fork was automatically filling out the Hawaii Request Access form: [seen here](https://github.com/CodeWithAloha/uipa/blob/c6e006c54dd13ba642661963a2d2bc46f9e2f73f/uipa_org/theme/data/Request-Access-form-12.1.15-fillable.docx).

This file is a `.docx` with certain template text in the file (ex: `[Agency_Name]`), this is replaced by what the user provides in a request and is then [attached to the request](https://github.com/CodeWithAloha/uipa/blob/c6e006c54dd13ba642661963a2d2bc46f9e2f73f/uipa_org/theme/models.py#L34).

Some questions:
- Do we need to update this file?
- Can this be done in a better way?

**Comment by russtoku** [comment]
> Do we need to update this file?

I don't think so. It's the official form (hasn't changed since being created) from OIP (Office of Information Practices, oip.hawaii.gov).

> Can this be done in a better way?

Perhaps but it's good enough. I think the main problem is the way that providing the reason for requesting a waiver of the fees is currently implemented. See https://github.com/CodeWithAloha/uipa/issues/61. It's confusing and apparently prone to errors.
